share/instruments/instruments.xml

```diff
@@ -6551,6 +6551,24 @@
                   <genre>concertband</genre>
                   <genre>marching</genre>
             </Instrument>
+            <Instrument id="bell-tree">
```

Comment:
Sorry, it's my fault this got in. I've removed this instrument from the spreadsheet for now. Could you drop this commit and then run `update_instruments_xml.py` and `lupdate` again in each subsequent commit where you ran them before? You can ignore any merge conflicts as these will disappear after you run the script to regenerate the files.

Reply:
Done!